README.md

```diff
@@ -8,19 +8,19 @@
 
 ### [](https://github.com/pushpinderpalsingh/SpaceDash#a-dashboard-for-space)A Dashboard for Space...🚀🚀🚀 
 
-SpaceDash is an app that **provides information** on all the present, past, and future **Launches and Vehicles of SpaceX**. The SpaceDash app also includes news and updates regarding vehicles and missions.
+SpaceDash is an app that provides information on all the present, past, and future **launches and vehicles of SpaceX**. The SpaceDash app also includes news and updates regarding all such vehicles and missions.
```

Comment:
I am happy with the initial changes to the first sentence, the second does not read well/is not grammatically correct. Please omit **such**.